### DISCUSSION TITLE
SrcBlock/NativeIndent demonstrates native indentation

### DISCUSSION THREAD

**Discussion by TobiasZawada** [emacsorphanage/adoc-mode#23] [Show and tell]
Hi @bbatsov,
the branch [SrcBlock/NativeIndent](https://github.com/bbatsov/adoc-mode/tree/SrcBlock/NativeIndent) demonstrates native indentation in code blocks by `indent-for-tab-command` in a corresponding language buffer.

**Comment by bbatsov** [comment]
Nice! Please, open a PR about it as well, so it's easier to leave feedback about it.

**Reply by TobiasZawada** [reply]
The source blocks themselves are not the problem. The problem is the other stuff.
And indentation is (pityingly) such an all-or-nothing-thing (`indent-line-function`).

On this field one needs to consider quite a lot of adoc-spec.
